src/fs/metadata_log_operations/unlink_or_remove_file.hh

```diff
@@ -150,7 +150,7 @@ class unlink_or_remove_file_operation {
                 return make_exception_future(no_more_space_exception());
             case metadata_log::append_result::APPENDED:
                 _metadata_log.memory_only_delete_dir_entry(*_dir_info, _entry_name);
-                _metadata_log.memory_only_delete_inode(_entry_inode);
+                _metadata_log.memory_only_delete_inode_after_dir_entry(_entry_inode);
```

Comment:
I think it should be done as call to one function if you need more context there. Just like memory_only_create_inode_as_dir_entry() is done.

Reply:
it is done this way because these calls are separated in the corresponding function in bootstrap

Reply:
I think it can be rewritten so that the operations there are called one after another, so that it will be possible to merge them.

Reply:
I think that commit that rewrites it (this bootstraping method) should go as 'FIXUP:' in this series, so that i can land in zpp_fs before this pull request. And another on top of that merges these functions together and this commit will stay here in this pull request. So 1. FIXUP commit that rewrites bootstraping method so that these calls are alongside each other. 2. commit that merges these calls into one.